WebClient/src/components/outsiderSuggestions/outsiderSuggestions.ts

```diff
@@ -168,26 +146,40 @@ export class OutsiderSuggestionsComponent {
         let kariquaCap = Math.ceil(((unbuffedBossHealth - 5) / -bubos - 1) / 0.5);
         let orphalasCap = Math.max(1, Math.ceil(((2 - unbuffedBossTimer) / chronos - 1) / 0.75)) + 2;
         let senakhanCap = Math.max(1, Math.ceil((100 / unbuffedTreasureChestChance) / (dora / 100 + 1) - 1));
-
-        let rhageistRatio = 0.125;
-        let kariquaRatio = 0.015;
-        let orphalasRatio = 0.035;
-        let senakhanRatio = 0.025;
+		
+        // Outsider Ratios
+        let rhageistRatio;
+        let kariquaRatio;
+        let orphalasRatio;
+        let senakhanRatio;
 
         if (ancientSouls < 100) {
             let ratioChange = ancientSouls / 100;
-            rhageistRatio *= ratioChange;
-            kariquaRatio *= ratioChange;
-            orphalasRatio *= ratioChange;
-            senakhanRatio *= ratioChange;
+            rhageistRatio = 0.2*ratioChange;
+            kariquaRatio = 0.01*ratioChange;
+            orphalasRatio = 0.05*ratioChange;
+            senakhanRatio = 0.05*ratioChange;
+        } else if (ancientSouls < 27000) {
+            rhageistRatio = 0.2;
+            kariquaRatio = 0.01;
+            orphalasRatio = 0.05;
+            senakhanRatio = 0.05;
+        } else if (ancientSouls < 60000) {
```

Comment:
Unless I can't read, I think that the ratios for `< 27000` and `< 60000` are the same (except for the `multiplier` which isn't used it seems)

Reply:
True. It's placeholder because I don't have spreadsheets for it yet. I can remove for now if you want.